packages/common/src/openshift/openshift.service.ts

```diff
@@ -35,8 +35,8 @@ namespace Online {
       if (this.is(HawtioMode.Cluster)) {
         const projects_client = this.K8SClientFactory.create(
           {
-           kind: KubernetesAPI.WatchTypes.PROJECTS,
-           labelSelector: _.get(configManager.config, "online.projectSelector", null),
+            kind: KubernetesAPI.WatchTypes.PROJECTS,
```

Comment:
This fails on vanilla Kubernetes as this translates to querying an OpenShift API. This must be changed to `WatchTypes.NAMESPACES` dynamically.